docs/odoo/ir.config_parameter.rst

```diff
@@ -0,0 +1,23 @@
+ir.config_parameter
+===================
+
+XML operations
+--------------
+
+Create new setting
+^^^^^^^^^^^^^^^^^^
+Code::
+
+        <record id="myid" model="ir.config_parameter">
+            <field name="key">auth_signup.allow_uninvited</field>
+            <field name="value">True</value>
+        </record>
+
+Search addons for *model="ir.config_parameter"* for more examples.
+
+Change existing setting
+^^^^^^^^^^^^^^^^^^^^^^^
+Code::
+
+    <function model="ir.config_parameter" name="set_param" eval="('auth_signup.allow_uninvited', True)" />
```

Comment:
Prons:
* works if you are not sure whether key already used or not

Cons:
* record is not deleted on uninstalling